static/js/vue-cdr-access/src/mixins/fileDownloadUtils.js

```diff
@@ -0,0 +1,43 @@
+import permissionUtils from "./permissionUtils";
+
+export default {
+    mixins: [permissionUtils],
+
+    data() {
+        return {
+            brief_object: this.briefObject || {}
+        }
+    },
+
+    computed: {
+        getOriginalFile() {
+            const original_file =  this.brief_object.datastream.find(file => file.startsWith('original_file'));
+            if (original_file === undefined) {
+                return undefined;
+            }
+
+            return original_file;
+        },
+
+        largestImageEdge() {
+            const file_info = this.getOriginalFile.split('|');
+            const edge_sizes = file_info[file_info.length - 1].split('x');
+            return edge_sizes[0] > edge_sizes[1] ? edge_sizes[0] : edge_sizes[1];
+        },
+
+        showImageDownload() {
+            return this.hasPermission(this.brief_object, 'viewAccessCopies') &&
```

Comment:
I think this should be viewOriginal since for the time being we are disallowing all the download options unless the user can access the original (which is why https://jira.lib.unc.edu/browse/BXC-4220 has been on hold while RCMT figures out what behavior we want)